README.md

```diff
@@ -1 +1,22 @@
-# 1807_colab
\ No newline at end of file
+# DWA Form Builder Tool
+The application will utilize the formTool to build html form elements. The formBuilder function within the formTool will accept an unlimited number of arguments to create form inputs, selects, labels and buttons. Sample arguments are instantiated in the index.js file and processed by the formTool before rendering the view. Navigation to index.html will display a form built with the formTool.
+
+### Installing
+
+1. clone or download repo
+2. navigate to root of app directory
+3. install dependencies `npm install`
+
```

Comment:
Documentation: By not providing examples of how to use the tool, what the inputs and outputs of it are you are severely limiting its usage by other devs and yourself, in several months.